support/acks.md

```diff
@@ -0,0 +1,76 @@
+---
+breadcrumb: Acknowledgements
+---
+# Acknowledgements
+
+The OpenSSL project depends on volunteer efforts and financial support
+from the end user community. That support comes in [many
+forms](donations.html).
+
+## <a href="current">Sponsorship Donations</a>
+
+We would like to identify and thank the following sponsors for their
+donations which give significant support to the OpenSSL project. Please
+note sponsors may choose to remain anonymous.
+
+------------------------------------------------------------------------
+
+<style type="text/css">
+  .sponsorlogo {
+  height: 100px !important;
+  width: 210px !important;
+  object-fit: contain !important;
+  object-position: 50% 50% !important;
+  padding-left: 15px !important;
+  padding-top: 0px !important;
+  padding-bottom: 0px !important;
+  padding-right: 15px !important;
+  }
+  .sponsorsection {
+  background-color: #ffffff !important;
+  padding-top: 10px !important;
+  text-align: center !important;
+  }
+</style>
+
+### Gold:
```

Comment:
Sponsors ought to be machine generated from a database of some kind.  Not for this PR I suspect.

Reply:
Not for this PR, indeed.